com.unity.render-pipelines.universal/Runtime/ScriptableRenderer.deprecated.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.ComponentModel;
+using UnityEngine.Scripting.APIUpdating;
+
+namespace UnityEngine.Rendering.Universal
+{
+    [MovedFrom("UnityEngine.Rendering.LWRP")] public abstract partial class ScriptableRenderer
+    {
+        // Deprecated in 10.x
+        [Obsolete("cameraDepth has been renamed to cameraDepthTarget. (UnityUpgradable) -> cameraDepthTarget")]
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public RenderTargetIdentifier cameraDepth
+        {
+            get => m_CameraDepthTarget;
+        }
+    }
+}
```

Comment:
We currently have two Deprecated.cs files (editor + runtime) and one for shaders. 
Would it make sense to move this there so we only have to look there for things to remove?

com.unity.render-pipelines.universal/Editor/Deprecated.cs
com.unity.render-pipelines.universal/Runtime/Deprecated.cs
com.unity.render-pipelines.universal/ShaderLibrary/Deprecated.hlsl

Reply:
ah good point. will do that.